includes/Data/Patterns.php

```diff
@@ -72,6 +72,30 @@ protected static function get_theme_step_patterns() {
 						'description' => __( 'A page for periodic news, announcements and ideas.', 'wp-module-onboarding' ),
 					),
 				),
+				'header-menu' => array(
```

Comment:
Can we maybe just have one key for header-menu somehow cause right now all the steps have one key and we are introducing inconsistency into the variable, maybe a single key with two subkeys?

Reply:
yup ... removed header-menu-body mapping .. was not needed